packages/mc-stats/src/class-mc-stats.php

```diff
@@ -0,0 +1,182 @@
+<?php
+/**
+ * Jetpack MC Stats package.
+ *
+ * @package  automattic/jetpack-mc-stats
+ */
+
+namespace Automattic\Jetpack;
+
+/**
+ * Class MC Stats, used to record stats using https://pixel.wp.com/g.gif
+ */
+class Mc_Stats {
```

Comment:
Could we do something like `A8c_Mc_Stats` or something to try to underscore this is for Automattic tracking only?